urbanaccess/config.py

```diff
@@ -42,13 +43,16 @@ class urbanaccess_config(object):
     logs_folder : str
         location to write log files
     log_file : bool
-        if true, save log output to a log file in logs_folder
+        if True, save log output to a log file in logs_folder
     log_console : bool
-        if true, print log output to the console
+        if True, print log output to the console
     log_name : str
         name of the logger
     log_filename : str
         name of the log file
+    txt_encoding : str
+        default encoding to use to read and write GTFS txt files. Must be
```

Comment:
To be as specific as possible, you could say something like "text encoding used by the GTFS files, to be passed to Python's open() function"

Reply:
agreed, fixed in https://github.com/UDST/urbanaccess/pull/80/commits/98aa4a743361f74f718da24874d9345a5807b1e1